org.lvnauth.LVNAuth.json

```diff
@@ -0,0 +1,41 @@
+{
+    "app-id": "org.lvnauth.LVNAuth",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "23.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "startup.sh",
+    "finish-args": [
+	    "--socket=x11",
+	    "--share=network",
+	    "--share=ipc",
+	    "--socket=pulseaudio",
+	    "--filesystem=home",
+	    "--device=dri"
+     ],
+    "modules": [
+        {
+            "name": "lvnauth",
+            "build-commands": [
+		            "install -D flatpak/startup.sh ${FLATPAK_DEST}/bin/startup.sh",
+                            "chmod +w ${FLATPAK_DEST}/bin/startup.sh",
```

Comment:
![image](https://github.com/flathub/flathub/assets/62639087/f79f85db-a9cf-45d7-9b61-e8b0df3fc9a1)

Something is wrong with your editor the indentation is getting messed up